instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -101,7 +101,9 @@ def span_attrs(kind, *args)
 
           def extract_operation(sql)
             # From: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/9244a08a8d014afe26b82b91cf86e407c2599d73/plugins/node/opentelemetry-instrumentation-pg/src/utils.ts#L35
-            sql.to_s.split[0].to_s.upcase
+            # Ignores prepend comment
+            comment_regex = %r{\A\/\*.*?\*\/}m
```

Comment:
Would it make sense to move this into constant?

Reply:
I concur with @simi 

@keiko713 Please extract this regex into a constant to avoid allocating it for every query.

Reply:
TIL Ruby freezes Regexes like it freezes strings. 

Running a file named `foo.rb` with the following ruby code:
```ruby
# frozen_string_literal: true

def foo
  %r{\A\/\*.*?\*\/}m.object_id
end

def bar
  'hello'.object_id
end

puts foo
puts foo
puts foo

puts bar
puts bar
puts bar
```

has this result:
```
➜  ~ ruby foo.rb
60
60
60
80
80
80
```

So I'm going to merge this in without moving it to a constant.